subscriber/src/subscriber/save.bal

```diff
@@ -87,26 +95,35 @@ function saveImagePdf(map<json> imageJson) {
     }
 }
 
-# Return the file name to store this result using the format:
```

Comment:
Shall we keep this function as is and rename it to `getPresidentialFileNameBase` and intro a new function `getParliamentaryFileNameBase` with just the PE logic? And set the function pointer in the main function?